department-of-veterans-affairs/caseflow
# Matthew roth1/appeals 7035

Resolves [APPEALS-7035](https://vajira.max.gov/browse/APPEALS-7035)

### Description
Add the column timeliness to the table judge_case_reviews

### Acceptance Criteria
- [ ] Value stored from the radio buttons for ‘Timely’ and ‘Untimely’ is available for future Tableau reporting and stored in the same table (judge_case_reviews) where other evaluation fields from the Evaluate Decision page are stored.
- [ ] Appeals that with decisions drafted and evaluated before the implementation of this new evaluation field, will have this value as NULL. Old cases will not be updated to reflect a value on this new evaluation field.

### Testing Plan
1. Switch to a Judge User (e.g. BVAABSHIRE) and navigate to their queue.
2. Locate an appeal that has a "Review" Task and click on the link to go to the Case Details Page.
3. In the Actions dropdown, select "Ready for Dispatch"
4. Select 'no special issues' and continue filling out the forms
5. When you get to the Evaluate Decision page, ensure that the "Was this decision timely?" Question and radio options appear in the Case Timeliness section.
6. Click 
7. Go into the rails console
8. Run command `ActiveRecord::Base.connection.execute("select * from judge_case_reviews where timeliness is not null").values`
9. Observe that a value of timely or untimely was stored for above review


### Database Changes
*Only for Schema Changes*


### Integrations: Adding endpoints for external APIs
* [ ] Integrations impacting functionality are tested in Caseflow UAT
